torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -612,7 +641,7 @@ def _partition_parameters(
                     params_sorted = sorted(param_group["params"], key=lambda t: t.numel(), reverse=True)
                     for param in params_sorted:
                         # Greedily add the parameter to rank with smallest size so far
-                        rank = sizes.index(min(sizes))
+                        rank = self._get_min_index(sizes)
```

Comment:
I needed `_get_min_index()` to implement the heuristic for approximating uniform bucket assignments, so I thought I might as well change this here as well to save one pass over the `sizes` list. Though, this will have negligible performance improvement since `len(sizes) == world_size`, which is not likely to be large.